tsk/fs/hfs.c

```diff
@@ -4234,9 +4234,7 @@ hfs_load_extended_attrs(TSK_FS_FILE * fs_file,
 
                 // Check the attribute fits in the node
                 //if (recordType != HFS_ATTR_RECORD_INLINE_DATA) {
-                if (attrFile.nodeSize < attributeLength + 2 + 16 + keyLength||
-
-                    attrFile.nodeSize >= 2 + 16 + keyLength + attributeLength + recOffset) {
+                if (attributeLength + 2 + 16 + keyLength > attrFile.nodeSize || recOffset + 2 + 16 + keyLength + attributeLength >= attrFile.nodeSize) {
```

Comment:
I always like using parens because not everybody knows the order of operations, and they are strangely different in differently languages. 

@uckelman-sf  - Why did you collapse this to one line?

Reply:
It had been one line before `4ff0ee390`. I was restoring it to how it was.

Reply:
Oh.